crates/next-custom-transforms/src/transforms/server_actions.rs

```diff
@@ -2096,15 +2108,23 @@ impl<C: Comments> VisitMut for ServerActions<C> {
         }
 
         if self.has_action || self.has_cache {
+            // Flip the map and convert it to a BTreeMap for deterministic
+            // ordering in the server references comment.
+            let export_names_ordered_by_reference_id: OrderedAtomMap = self
+                .reference_ids_by_export_name
+                .iter()
+                .map(|(export_name, reference_id)| (reference_id.clone(), export_name.clone()))
+                .collect();
```

Comment:
```suggestion
            let export_names_ordered_by_reference_id = self
                .reference_ids_by_export_name
                .iter()
                .map(|(export_name, reference_id)| (reference_id.clone(), export_name.clone()))
                .collect::<BTreeMap<_, _>>();
```
a bit overkill to have the `OrderedAtomMap` type alias for basically one usage of it.

Reply:
and you don't have to `clone()` here technically, you should be able to just store `&Atom` in the tree map.